ci/dhis2-netlify-deploy-pr.yml

```diff
@@ -17,14 +17,14 @@ concurrency:
     cancel-in-progress: true
 
 jobs:
-    preview:
+    deploy:
         runs-on: ubuntu-latest
         if: "!github.event.push.repository.fork && github.actor != 'dependabot[bot]'"
         steps:
             - uses: actions/checkout@v2
             - uses: actions/setup-node@v1
               with:
-                  node-version: 12.x
+                  node-version: 14.x
```

Comment:
We're using node 16 above, should we use the same version?

Reply:
Yeah you're right. 16 or even 18 would be good. I'll update that.